tests/data/mmd_strict.xsd

```diff
@@ -142,6 +138,11 @@
         <xs:sequence>
             <xs:element name="identifier" type="mmd:use_constraint_identifier_type" maxOccurs="1" minOccurs="0"></xs:element>
             <xs:element name="resource" type="mmd:use_constraint_resource_type" maxOccurs="1" minOccurs="0"></xs:element>
+            <xs:element name="license_text" type="xs:string" maxOccurs="1" minOccurs="0">
+                <xs:annotation>
+                    <xs:documentation>This element holds the a free-text non formal license. If used identifier and resource should be left empty.</xs:documentation>
```

Comment:
Spelling error

Reply:
yes, seems so..

Reply:
but this is just a file for the tests, so it's not important. The same error is probably in mmd though, so that could be fixed..